content/docs/pulumi-cloud/deployments/api.md

```diff
@@ -380,6 +381,24 @@ The GitHub block describes settings for Pulumi Deployments' GitHub integration.
 }
 ```
 
+### CacheOptions
+
+The Cache Options block defines settings related to dependency caching during Deployments. Using dependencies from the cache makes deployments significantly faster. Learn more about dependency caching on the [FAQ page](/docs/pulumi-cloud/deployments/faq/#dependency-caching).
+
+This option is only available for Pulumi-managed deployment agents.
+
+* **enable** (boolean): Whether to use Dependency Caching.
```

Comment:
it's currently `enabled` but I can change it to `enable`. Do we have a preference? ([ref](https://github.com/pulumi/pulumi-pulumiservice/pull/436/files#diff-6cedc86d3dd0f04d7f5be7d3c7bab9ccf59bbebc0ccbbe412bebb4f36456ad55R560))

Reply:
It should match the API, [which is `enable`](https://github.com/pulumi/pulumi-service/blob/a0ad38f3f3c7e27de2a697bdfc111c634f564ad9/cmd/console2/src/api/types/deployment.ts#L325)